molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl

```diff
@@ -144,8 +135,7 @@
     </#if>
 
 <#-- Start application content -->
-<div class="container-fluid mg-page-content"
-     style="padding-top: <#if app_settings.logoTopHref?? && (!version?? ||version == 1)>${app_settings.logoTopMaxHeight + 60}<#else>60</#if>px;">
+<div class="container-fluid mg-page-content">
```

Comment:
Will this be fixed in the theme?

Reply:
No, I missed this. Bootstrap 3 version fails. Will revert some changes

Reply:
Fixed